Test2/Tests/SettingPageBase.cs

```diff
@@ -0,0 +1,150 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+
+namespace Test2.Tests
+{
+
+    internal class SettingPage
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            driver.Url = "https://localhost:5001/Settings";
+        }
+        [TearDown]
```

Comment:
Please add an empty line between methods.
```suggestion
        }
        
        [TearDown]
```